llvmlite/tests/test_binding.py

```diff
@@ -269,7 +269,7 @@ def test_set_option(self):
 
     def test_version(self):
         major, minor, patch = llvm.llvm_version_info
-        self.assertIn((major, minor), [(3, 6), (3, 7)])
+        self.assertIn((major, minor), [(3, 7), (3, 8)])
```

Comment:
Le 12/07/2016 19:50, Stan Seibert a écrit :
> 
> Shouldn't this always be 3.8?

Yes... originally I thought llvmlite could be compatible with
consecutive LLVM versions (3.6 and 3.7, at the time), but I gave up on
the idea.